### PR TITLE
[Meetings][Updatedin-meetingdialog][1609291]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added information about in-meeting dialog and task module for issue 2870. Grammar and language changes. Please review.

### CODE REVIEW

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 1]
```diff
@@ -251,74 +251,88 @@ The meetings app capabilities are declared in your app manifest through the **co
...
 
-✔ Use the [NotificationSignal API](create-apps-for-teams-meetings.md#notificationsignal-api) to signal that a bubble notification needs to be triggered.
+#### In-meeting dialog
 
-✔ As part of the notification request payload, include the URL where the content to be showcased is hosted.
+The in-meeting dialog box can be used to engage participants during the meeting and collect information or feedback during the meeting. Use the [`NotificationSignal`](/graph/api/resources/notifications-api-overview?view=graph-rest-beta&preserve-view=true) API to signal that a bubble notification must be triggered. As part of the notification request payload, include the URL where the content to be shown is hosted.
 
-✔ In-meeting dialog must not use task module.
+In-meeting dialog must not use task module. Task module is not invoked in meeting chat rather external resource URL is used to display content bubble pop-up in meeting. Then you can use `submitTask` method to submit the data in meeting chat.
```
Comment (Trinetra-MSFT) [inline msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md:316]: Can you rephrase this sentence properly?

**Review by Trinetra-MSFT — CHANGES_REQUESTED** [review]

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 2]
```diff
@@ -10,51 +10,53 @@ keywords: teams apps meetings user participant role api
...
+|**participantId**| string | Yes | The participant ID is the user ID. It is available in Tab SSO, Bot Invoke, and Teams Client SDK. It is recommended to get a participant ID from the Tab SSO. |
+|**tenantId**| string | Yes | The tenant ID is required for the tenant users. It is available in Tab SSO, Bot Invoke, and Teams Client SDK. It is recommended to get a tenant ID from the Tab SSO. |
 
 #### Example
 
-# [C#/.NET](#tab/dotnet)
+# [C#](#tab/dotnet)
 
 ```csharp
 protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
```
Comment (Trinetra-MSFT) [inline msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md:62]: Document doesn't talk about from where we will get the meeting id and participants id

**Review by Trinetra-MSFT — CHANGES_REQUESTED** [review]

**Review by Trinetra-MSFT — APPROVED** [review]
Looks Good

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR
